tests/foreman/remote_DB/test_remote_DB.py

```diff
@@ -0,0 +1,228 @@
+"""
+
+from robottelo.decorators import stubbed, tier1
+from robottelo.test import TestCase
+
+
+class RemoteDBTestCase(TestCase):
+    @stubbed()
+    @tier1
+    def test_install_external_foreman(self):
+        """Install Satellite with external foreman database
+
+        :id: 1f6c9243-7157-4bff-bd78-26fa4a6a55fb
+
+        :Steps: run satellite-installer with foreman-db options set
+
+        :expectedresults: Completed Satellite installation
+
+        :CaseLevel: Integration
+
+        :CaseComponent: Installer
+        """
+
+    @stubbed()
+    @tier1
+    def test_install_external_candlepin(self):
+        """Install Satellite with external candlepin database
+
+        :id: 2da6173c-ba16-4130-8992-35d3daa40ab2
+
+        :Steps: run satellite-installer with candlepin-db options set
+
+        :expectedresults: Completed Satellite installation
+
+        :CaseLevel: Integration
+
+        :CaseComponent: Installer
+        """
+
+    @stubbed()
+    @tier1
+    def test_install_external_pulp(self):
+        """Install Satellite with external pulp database
+
+        :id: 90347920-d183-487e-9b65-03f6d581a173
+
+        :Steps: run satellite-installer with pulp-db options set
+
+        :expectedresults: Completed Satellite installation
+
+        :CaseLevel: Integration
+
+        :CaseComponent: Installer
+        """
+
+    @stubbed()
+    @tier1
+    def test_install_external_all_with_SSL(self):
+        """Install Satellite with all external databases with SSL
+
+        :id: 5b4fd17a-7562-4d25-94e2-cda8f088a949
+
+        :Steps: Run satellite-installer with pulp, foreman and candlepin db options set
+
+        :expectedresults: Successful installation, all services running
+
+        :CaseLevel: System
+
+        :CaseComponent: Installer
+        """
+
+    @stubbed()
+    @tier1
+    def test_reset_satellite_installation_external_all_with_SSL(self):
+        """Reset Satellite installation with all ext SSL
+
+        :id: b8f3752e-d59f-40c5-9ecf-42a1dd1dca12
+
+        :Steps:
+
+            1. Run satellite-installer with pulp, foreman and candlepin db options set
+            2. Run satellite-installer --reset
+
+        :expectedresults:
+
+            1. Successful installation, all services running
+            2. Successful reset, all services running
+
+        :CaseLevel: System
+
+        :CaseComponent: Installer
+        """
+
+    @stubbed()
+    @tier1
+    def test_offline_backup_external_all_with_SSL(self):
+        """Perform offline backup of external database
+
+        :id: dc452b72-66b8-4a69-adb1-da089a0d4942
+
+        :Steps: Run foreman-maintain backup offline
+
+        :expectedresults: Database backup created
+
+        :CaseLevel: System
+        """
+
+    @stubbed()
+    @tier1
+    def test_online_backup_external_all_with_SSL(self):
+        """Perform online backup of external database
+
+        :id: 2781aeb7-742e-4c95-ad57-c2d1c025bf90
+
+        :Steps: Run foreman-maintain backup online
```

Comment:
I will vote for all these test cases related to foreman-maintain should got to https://github.com/SatelliteQE/testfm as we have all automation there. Rather than putting manualonly in robottelo, you can run testfm tests on external db setup.